src/plugins/intel_npu/tests/unit/npuw/unpack.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#if 0
+#include "unpack.hpp"
```

Comment:
this may trigger a warning but let's see

Reply:
Done. Added `#ifdef __AVX2__`